Documentation/writing-principles-and-guidelines/spelling-and-typo3-terminology.rst

```diff
@@ -1,111 +1,1006 @@
+        management and invalidation of cached data.
+
+    ..  _certification--t-cc--t-cd--t-ci--t-ce:
+    -   *Certification (TCCC, TCCD, TCCI, TCCE)*
+
+        Certifications in the TYPO3 ecosystem, such as TCCC (Consultant), TCCD
+        (Developer), TCCI (Integrator), and TCCE (Editor) confirm the
+        proficiency of developers and integrators in various aspects of TYPO3
+        CMS. TYPO3 has an official certification strategy.
+
+    ..  _cig-sig--special-interest-group:
+    -   *CIG/SIG (Special Interest Group)*
+
+        Special Interest Groups (SIGs) are groups of experts and enthusiasts who
+        focus on specific topics within the TYPO3 ecosystem and work on
+        improving those areas.
+
+    ..  _clipboard:
+    -   *Clipboard*
+
+        The clipboard in the TYPO3 backend is a tool for copying, cutting, and
+        pasting content elements and records.
+
+    ..  _colpos:
+    -   *colPos*
+
+        :spl:`colPos` is a column in the TYPO3 database that defines the
+        position and layout of content elements on a page within a template.
+
+    ..  _constants-setup:
+    -   *Constants/Setup*
+
+        Constants and Setup are configuration options in TYPO3 TypoScript that
+        set basic settings and variables for the website. "Constants" can be
+        regarded as variables, referencing contents defined in the backend GUI.
+        The "Setup" uses these "Constants" to put the variables to the place
+        they are needed, to define behaviour of the frontend (and sometimes also
+        backend).
+
+    ..  _content-blocks:
+    -   *Content Blocks*
+
+        Content blocks are predefined layouts and content elements that can be
+        used to create page content in the TYPO3 backend. Currently it refers to
+        an extension, whose feature set shall be part of TYPO3 v13. Content
+        Blocks are small sets of configuration, defining both Backend input and
+        frontend output.
+
+    ..  _content-elements:
+    -   *Content Elements*
+
+        Content elements in TYPO3 are building blocks of content that can shown
+        in the frontend. Each content element has many (and also custom)
+        attributes, even consisting of nested hierachies of further content
+        elements.
+
+    ..  _core:
+    -   *Core*
+
+        The TYPO3 Core is the central framework of the CMS that provides all
+        basic functions and features.
+
+    ..  _core-development:
+    -   *Core Development*
+
+        Core Development refers to the development and maintenance of the
+        central TYPO3 framework by the Core Team.
+
+    ..  _core-merger:
+    -   *Core Merger*
+
+        A Core Merger is a person or team member responsible for merging code
+        changes and updates into the TYPO3 core. TYPO3 Core Mergers are elected
+        in a formal process.
+
+    ..  _core-team:
+    -   *Core Team*
+
+        The Core Team consists of the main developers and contributors
```

Comment:
Why are Core Mergers not part of that group?

Reply:
They are, "main developers" would include that. Will rephrase.